app/controllers/api/v1/sessions_controller.rb

```diff
@@ -10,6 +10,12 @@ class API::V1::SessionsController < Devise::SessionsController
   # The #create method (through the /api/v1/users/sign_in route) receives:
   # {"user"=>{"login"=>"login", "password"=>"password"}}
   #
+  # It return a json object with a success message, and a path which the user should
+  # be redirected to
+  #
+  # this method also accepts an additional top level json paramter "after_sign_in_path"
```

Comment:
actually no it means it can be passed this parameter. I'll update the comment to try to make it more clear.